src/forms/FormActions.js

```diff
@@ -361,9 +361,21 @@ export const fetchSubmissions = (formId, page = 0) => {
   };
 };
 
-export const updateSubmission = props => dispatch => {
-  dispatch(updateActiveSubmission(props));
-  // TODO: go to server when API is done
+export const updateSubmissionFlags = props => (dispatch, getState) => {
+  const state = getState();
+  const { activeSubmission } = state.forms;
+  const keys = Object.keys(props);
+  const allKeys = keys.concat(state.forms[activeSubmission].flags);
+
+  dispatch(updateActiveSubmission({
+    // remove the flags that are for removing
+    flags: [...new Set(allKeys.filter(k => props[k] !== false))]
```

Comment:
this is all really good, but could you sprinkle some comments in this section so we're not confused when we come back to this in a couple of months?